[sdformat] fatal error: urdf_model/model.h: No such file or directory

while building `sdformat` I get the following error:
```
[ 52%] Building CXX object src/CMakeFiles/sdformat9.dir/Types.cc.o
/home/arch/.cache/pikaur/build/sdformat/src/sdformat-sdformat9_9.2.0/src/parser_urdf.cc:30:10: fatal error: urdf_model/model.h: No such file or directory
   30 | #include <urdf_model/model.h>
      |          ^~~~~~~~~~~~~~~~~~~~
```
I have `urdf_model/model.h` already installed in `/opt/ros2/`. Do I have to point there?

--- a/sdformat/.SRCINFO
+++ b/sdformat/.SRCINFO
@@ -15,7 +15,7 @@ pkgbase = sdformat
 	depends = tinyxml
 	depends = ignition-math>=6
 	depends = python-psutil
-	optdepends = urdfdom
+	depends = urdfdom
 	source = https://github.com/osrf/sdformat/archive/sdformat9_9.2.0.tar.gz
 	sha256sums = 0e42001d92aa2c089c7d0c4ea6a30db2beeff0af3a9a357e7ccd0a4e1131cae7
 

--- a/sdformat/.SRCINFO
+++ b/sdformat/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = sdformat
 	pkgdesc = SDF Converter for gazebo
 	pkgver = 9.2.0
-	pkgrel = 1
+	pkgrel = 2
 	url = http://sdformat.org/
 	arch = i686
 	arch = x86_64

--- a/sdformat/PKGBUILD
+++ b/sdformat/PKGBUILD
@@ -6,8 +6,7 @@ pkgdesc="SDF Converter for gazebo"
 arch=('i686' 'x86_64')
 url="http://sdformat.org/"
 license=('Apache')
-depends=('boost' 'tinyxml' 'ignition-math>=6' 'python-psutil')
-optdepends=('urdfdom')
+depends=('boost' 'tinyxml' 'ignition-math>=6' 'python-psutil' 'urdfdom')
 makedepends=('cmake' 'doxygen' 'ignition-cmake' 'ignition-tools' 'ruby>=1.9.1')
 source=("https://github.com/osrf/${pkgname}/archive/${pkgname}9_${pkgver}.tar.gz")
 sha256sums=('0e42001d92aa2c089c7d0c4ea6a30db2beeff0af3a9a357e7ccd0a4e1131cae7')

--- a/sdformat/PKGBUILD
+++ b/sdformat/PKGBUILD
@@ -1,7 +1,7 @@
 #Maintainer: acxz <akashpatel2008 at yahoo dot com>
 pkgname=sdformat
 pkgver=9.2.0
-pkgrel=1
+pkgrel=2
 pkgdesc="SDF Converter for gazebo"
 arch=('i686' 'x86_64')
 url="http://sdformat.org/"

